No numpy in release_merge_info environment

From:

https://github.com/sot/chandra_limits/actions/runs/6124534626/job/16624738268

```
File "/usr/local/bin/skare3-release-merge-info", line 5, in <module>
    from skare3_tools.github.scripts.release_merge_info import main
  File "/usr/local/lib/python3.10/dist-packages/skare3_tools/github/scripts/release_merge_info.py", line 14, in <module>
    import numpy as np
ModuleNotFoundError: No module named 'numpy'
Error: Process completed with exit code 1.
```

This is probably more a skare3 issue than a skare3_tools issue but figured I'd put it someplace.

--- a/requirements.txt
+++ b/requirements.txt
@@ -6,3 +6,5 @@ pyyaml
 gitpython
 packaging
 numpy
+json5
+keyring

--- a/requirements.txt
+++ b/requirements.txt
@@ -5,3 +5,4 @@ jinja2
 pyyaml
 gitpython
 packaging
+numpy

